oap-server/server-storage-plugin/storage-influxdb-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/influxdb/base/InfluxStorageDAO.java

```diff
@@ -53,6 +53,6 @@ public IRecordDAO newRecordDao(StorageBuilder<Record> storageBuilder) {
 
     @Override
     public INoneStreamDAO newNoneStreamDao(StorageBuilder<NoneStream> storageBuilder) {
-        return new NoneStreamDAO();
+        return new NoneStreamDAO(influxClient, storageBuilder);
```

Comment:
`INoneStreamDAO` is used for web interactive job, such as @mrproliu 's profiling task. Basically, this should have little data. Is the influxDB suitable for this?

Reply:
That sounds the relationship database better.

Reply:
There would be much data, just easy to query should be enough. Anyway, based on your decision.